nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2944,95 +3048,95 @@ at this time.
+|`nifi.provenance.repository.directory.default`*|The location of the Provenance Repository. The default value is `./provenance_repository`. +
  +
 	*NOTE*: Multiple provenance repositories can be specified by using the *_nifi.provenance.repository.directory._* prefix with unique suffixes and separate paths as values. +
  +
 	For example, to provide two additional locations to act as part of the provenance repository, a user could also specify additional properties with keys of: +
  +
-	nifi.provenance.repository.directory.provenance1=/repos/provenance1 +
-	nifi.provenance.repository.directory.provenance2=/repos/provenance2 +
+	`nifi.provenance.repository.directory.provenance1`=/repos/provenance1 +
+	`nifi.provenance.repository.directory.provenance2`=/repos/provenance2 +
  +
 	Providing three total locations, including `nifi.provenance.repository.directory.default`.
-|nifi.provenance.repository.max.storage.time|The maximum amount of time to keep data provenance information. The default value is `24 hours`.
-|nifi.provenance.repository.max.storage.size|The maximum amount of data provenance information to store at a time.
+|`nifi.provenance.repository.max.storage.time`|The maximum amount of time to keep data provenance information. The default value is `24 hours`.
+|`nifi.provenance.repository.max.storage.size`|The maximum amount of data provenance information to store at a time.
 	The default value is `1 GB`. The Data Provenance capability can consume a great deal of storage space because so much data is kept.
 	For production environments, values of 1-2 TB or more is not uncommon. The repository will write to a single "event file" (or set of
 	"event files" if multiple storage locations are defined, as described above) for some period of time (defined by the
-	nifi.provenance.repository.rollover.time and  nifi.provenance.repository.rollover.size properties). Data is always aged off one file at a time,
+	`nifi.provenance.repository.rollover.time` and `nifi.provenance.repository.rollover.size` properties). Data is always aged off one file at a time,
 	so it is not advisable to write to a single "event file" for a tremendous amount of time, as it will prevent old data from aging off as smoothly.
-|nifi.provenance.repository.rollover.time|The amount of time to wait before rolling over the "event file" that the repository is writing to.
-|nifi.provenance.repository.rollover.size|The amount of data to write to a single "event file." The default value is `100 MB`. For production
+|`nifi.provenance.repository.rollover.time`|The amount of time to wait before rolling over the "event file" that the repository is writing to.
+|`nifi.provenance.repository.rollover.size`|The amount of data to write to a single "event file." The default value is `100 MB`. For production
 	environments where a very large amount of Data Provenance is generated, a value of 1 GB is also very reasonable.
```

Comment:
The value `1 GB` should be code formatted.

Reply:
I'm not against this change.  I left ones like this alone, because it seemed like only the default values were code formatted.

Reply:
I understand why that might be the existing case. I think in general if it's a value that should be typed in literally, it should be `code formatted`.

Reply:
Sounds good.